deploy/scripts/teams/team.template

```diff
@@ -0,0 +1,23 @@
+parameters:
+  linux:
+    system:
+      user:
+        ${SNAME}:
+          enabled: true
+          name: ${SNAME}
+          sudo: true
+          full_name: ${FNAME}
+          home: /home/${SNAME}
+          email: ${EMAIL}
+  openssh:
+    server:
+      enabled: true
+      user:
+        ${SNAME}:
+          enabled: true
+          public_keys: ${public_keys:${SNAME}}
+          user: ${linux:system:user:${SNAME}}
+  public_keys:
```

Comment:
IMHO not needed, add public keys directly under user:${NAME}:public_keys to save some lines.